js/piwik.js

```diff
@@ -4962,14 +5001,23 @@ if (typeof Piwik !== 'object') {
                 /**
                  * Set heartbeat (in seconds)
                  *
-                 * @param int minimumVisitLength
-                 * @param int heartBeatDelay
+                 * @param int heartBeatDelay Defaults to 15.
                  */
-                setHeartBeatTimer: function (minimumVisitLength, heartBeatDelay) {
-                    var now = new Date();
+                setHeartBeatTimer: function (heartBeatDelay) {
```

Comment:
Does this break API? Removing the first parameter? I reckon it was not much used before? Should we mention it in changelog? Or maybe we do already? :)